ad-click/ad-click-impl/src/test/resources/maestro_tests/ad_click_detection_flows/10_-_m.js_bing-provided;_ad_domain_provided_but_incorrect_(dsl_not_needed).yaml

```diff
@@ -32,27 +32,27 @@ appId: com.duckduckgo.mobile.android
 - assertVisible:
     text: "Publisher site"
 - tapOn:
-    id: "com.duckduckgo.mobile.android:id/privacyGradeButton"
+    id: "com.duckduckgo.mobile.android:id/omnibarIconContainer"
 - assertVisible:
     text: "Requests Blocked from Loading"
 - tapOn:
     text: "Requests Blocked from Loading"
 - assertVisible:
-    text: "The following third-party domains' requests were blocked from loading because they were identified as tracking requests. If a company's requests are loaded, it can allow them to profile you."
+    text: "The following third-party domains’ requests were blocked from loading because they were identified as tracking requests. If a company's requests are loaded, it can allow them to profile you."
 - assertVisible:
     text: "About our Web Tracking Protections"
 - assertVisible:
-    text: "Ad Company"
+    text: ".*Ad Company"
 - assertVisible:
     text: "convert.ad-company.site"
 - assertVisible:
-    text: "ad-company.site"
+    text: "www.ad-company.site"
```

Comment:
This will be reverted (and all instances) once it's fixed in https://app.asana.com/0/0/1203090393145429/f